topics-and-advance-readings/trust-ux-issues.adoc

```diff
@@ -1,16 +1,41 @@
-= The Threat Model
+= UX Issues in Communicating Threats and Countermeasures
 Genevieve Primavera <https://github.com/genevievePrima>
 v0.1
 :description: Rough draft of topic paper for Rebooting Web of Trust.
 
-The biggest security issue facing Internet users is their own lack of awareness that they even need it in the first place. And if they do have this awareness, many and most don’t even know where to start. So what we have is the larger population of people accepting their lack of security and privacy in order to continue to communicate, take part in commerce, and thrive in the modern world. This chosen mantra of “ignorance is bliss,” further puts the common user in the dark when it comes to security, until she becomes a victim of it, and it’s too late.
+by Genevieve Primavera https://github.com/genevievePrima[@genevievePrima] and Sean Gilligan https://github.com/msgilligan[@msgilligan]
```

Comment:
@genevievePrima I decided to link to our Github addresses, but we could do email if you prefer.